### PR TITLE
OEL-3318: Prepare for php8.3

### PR DESCRIPTION
**Pull request by drishu**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,15 +14,11 @@
         "nuvoleweb/robo-config": "^3.1"
     },
     "require-dev": {
-        "openeuropa/code-review-library": "1.x-dev",
-        "phpunit/phpunit": "^9.4",
+        "openeuropa/code-review-library": "^1.0.0-alpha1",
```
Comment (AaronGilMartinez) [inline composer.json:17]: Set minimum stability to alpha again

**Review by AaronGilMartinez — CHANGES_REQUESTED** [review]

**Comment by sergepavle** [comment]
Looks like now we can cleanup in composer.json values of 'config.allow-plugins' with this:
`        "allow-plugins": {
            "dealerdirect/phpcodesniffer-composer-installer": true,
            "phpro/grumphp-shim": true
        }`

File: composer.json [hunk 2]
```diff
@@ -14,15 +14,11 @@
...
+        "phpunit/phpunit": "^9.5",
         "slevomat/coding-standard": "^7.0 || ^8.0"
```
Comment (sergepavle) [inline composer.json]: Do we still need this package? I didn't find anywhere in composer.lock other packages that used this.

**Review by sergepavle — DISMISSED** [review]

File: composer.json [hunk 3]
```diff
@@ -47,9 +42,6 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "cweagans/composer-patches": true,
```
Comment (AaronGilMartinez) [inline composer.json:50]: There is some leftover configuration for patches in extra

**Review by AaronGilMartinez — CHANGES_REQUESTED** [review]

**Review by AaronGilMartinez — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**